clickhouse/columns/tuple.h

```diff
@@ -37,7 +37,7 @@ class ColumnTuple : public Column {
     size_t Size() const override;
 
     /// Makes slice of the current column.
-    ColumnRef Slice(size_t, size_t) const override { return ColumnRef(); }
+    ColumnRef Slice(size_t, size_t) const override;
```

Comment:
Please add some unit tests that validate implementation of `ColumnTuple::Slice()`, both positive and negative cases.